### PR TITLE
Kafka arbeidsplassen

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

**Comment by stianStensli** [comment]
I think we should write a simple test for the new kafka set up

**Comment by stianStensli** [comment]
Maybe we should get Alexander Gård to approve? 🤷‍♂️

File: src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        if (melding.getSlettetDato() != null) {
+            cvRepository.upsert(aktoerId, false);
+            elasticServiceV2.updateHarDeltCv(aktoerId, false);
+        } else {
+            cvRepository.upsert(aktoerId, true);
+            elasticServiceV2.updateHarDeltCv(aktoerId, true);
+        }
+    }
+
+    public void setCVSamtykke(AktorId aktoerId) {
```
Comment (stianStensli) [inline src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java:37]: Denne kan fjernes

File: src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    public void behandleKafkaMelding(CVMelding melding) {
+        AktorId aktoerId = melding.getAktoerId();
+
+        if (melding.getRessurs() != CV_HJEMMEL) {
+            log.info("Ignorer melding for ressurs {} for bruker {}", melding.getRessurs(), aktoerId);
+            return;
+        }
+
+        if (melding.getSlettetDato() != null) {
```
Comment (stianStensli) [inline src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java:28]: maybe flip this if logic? "stay positive"

Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java:28]: Done :)

File: src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java (hunk 1 above)
Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java:37]: Denne er brukt fra AdminController

File: src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java (hunk 2 above)
Comment (stianStensli) [reply src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java:28]: It does not appear to have changed

Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/cv/CVServiceFromAiven.java]: Just pushed that change..

**Review by stianStensli — APPROVED** [review]

**Pull request merged**